package.json

```diff
@@ -9,9 +9,10 @@
   "homepage": "http://react-bootstrap.github.io/",
   "scripts": {
     "build": "babel-node tools/build-cli.js",
+    "lib-build": "babel-node tools/build-cli.js --lib-only",
     "test-watch": "karma start",
-    "test-coverage": "npm run build && COVERAGE=true karma start --single-run",
-    "test": "npm run lint && npm run build && karma start --single-run && _mocha --compilers js:babel-core/register test/server/*Spec.js",
+    "test-coverage": "npm run lib-build && COVERAGE=true karma start --single-run",
+    "test": "npm run lint && npm run lib-build && karma start --single-run && _mocha --compilers js:babel-core/register test/server/*Spec.js",
```

Comment:
Since this command is the only one invoked on CI i liked that it at least built the docs and bower release to make sure it still works and is not broken.